feat(client-presence): Expose `Presence` at Workspaces and State objects (#24396)
microsoft/FluidFramework: .changeset/giant-news-appear.md

---
"@fluidframework/presence": minor
"__section": other
---

> Expose `Presence` at Workspaces and State objects

Reviewer: ```suggestion
Presence object is now accessible from Workspaces and State objects
```

> Users can now access Presence through `.presence()` all Workspaces and State objects:

Reviewer 2: ```suggestion
Users can now access `Presence` through `.presence` in all Workspaces and State objects:
```

> Users can now access `Presence` through `.presence` in all Workspaces and State objects:

Reviewer: ```suggestion
Users can now access the `Presence` object through `.presence` on all Workspaces and State objects:
```
